Header header
string robot_id
float64 x
float64 y
int32 step_number
bool next_step
bool consensus
bool move